### ISSUE TITLE
Segmentation fault on loading tessellation geometry

### ISSUE THREAD

**Issue by Moult** [IfcOpenShell/IfcOpenShell#814]
Tested using latest IfcOpenBot build.

[MaintenanceZoneIFC4_ver2.ifc.txt](https://github.com/IfcOpenShell/IfcOpenShell/files/4377928/MaintenanceZoneIFC4_ver2.ifc.txt)

Steps to reproduce:

```
import ifcopenshell
import ifcopenshell.geom
f = ifcopenshell.open('MaintenanceZoneIFC4_ver2.ifc')
s = ifcopenshell.geom.settings()
i = ifcopenshell.geom.iterator(s, f)
v = i.initialize()
... segmentation fault
```

I tried using gdb but it seems the IfcOpenBot builds don't have debugging symbols.

**Comment by Moult** [comment]
Cheers :) Does the commit fix it?

**Comment by aothms** [comment]
Hope so ;) new IfcOpenBot builds underway

**Comment by Moult** [comment]
https://github.com/IfcOpenBot/IfcOpenShell/commit/ed989836872b137044d62e7674eb00b74fcee26d - the IfcOpenBot build seems to be missing Blender Python 3.7 for Windows? What happened? I didn't realise Blender had moved to Python 3.8?

**Comment by aothms** [comment]
Not sure what happened there :( I'll look into it.

Op vr 27 mrt. 2020 om 10:26 schreef Dion Moult <notifications@github.com>:

**Comment by Moult** [comment]
By the way, the latest build also is missing Blender Python 3.7 for Windows: https://github.com/IfcOpenBot/IfcOpenShell/commit/c99bf6e63a30e2883f93b41df9198574e17df6fc

**Comment by aothms** [comment]
> latest build also is missing Blender Python 3.7 for Windows

Fixed, new builds underway for 0.6 as well

https://github.com/IfcOpenBot/IfcOpenShell/commit/8625aab5f0c2c4752eb10ad20daee744547b377e#commitcomment-38127532

**Comment by Moult** [comment]
Confirming fixed!
